controllers/cinder_controller.go

```diff
@@ -627,6 +624,8 @@ func (r *CinderReconciler) reconcileNormal(ctx context.Context, instance *cinder
 		}
 	}
 
+	instance.Status.Conditions.MarkTrue(condition.NetworkAttachmentsReadyCondition, condition.NetworkAttachmentsReadyMessage)
```

Comment:
I'm confused. Why is the top cinder_controller concerning itself with CinderAPI's network attachments (and only CinderAPI's attachments), when the cinderapi_controller has its own code?

Reply:
The comment on L610 explains this controller looks at network attachments because dbSync needs the same ones as CinderAPI.